stablehlo/integrations/python/tests/stablehlo_to_tf_saved_model_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 The StableHLO Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import mlir.dialects.stablehlo as stablehlo
+import mlir.ir as ir
+import numpy as np
+from savedmodel.stablehlo_to_tf_saved_model import InputLocation, stablehlo_to_tf_saved_model
```

Comment:
This still feels strange for `pip install stablehlo` to include a top-level `savedmodel` import? Can we move this to a `stablehlo` subdirectory at least?

Reply:
PTAL at https://github.com/openxla/stablehlo/pull/2157#discussion_r1556553712